src/main/resources/assets/paths/user.yaml

```diff
@@ -34,9 +34,6 @@ put:
             displayName:
               description: The display name for the user
               type: string
-            institutionId:
```

Comment:
If this was using openapi generated code, there would be a different object type for this request (vs the admin API) so we wouldn't need to add a specific validation for this field.

We could do this here too, e.g., have both `SelfUserUpdateFields` and `AdminUserUpdateFields`. Then the `400` would be thrown when `gson.fromJson()` is called.